From 6d246a5d9f2f93d4b3297c96301c7d0b5fbb59f4 Mon Sep 17 00:00:00 2001
From: DrRac27 <drrac27@riseup.net>
Date: Fri, 4 Feb 2022 23:40:04 +0100
Subject: [PATCH] Add X-Purism-FormFactor to desktop file

---
 packaging/moment.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packaging/moment.desktop b/packaging/moment.desktop
index 5f0fa101..cfcfbdf4 100644
--- a/packaging/moment.desktop
+++ b/packaging/moment.desktop
@@ -7,3 +7,4 @@ Icon=moment
 Terminal=false
 Categories=Network;Chat;InstantMessaging;Qt;
 StartupWMClass=moment
+X-Purism-FormFactor=Workstation;Mobile;
-- 
GitLab

